src/src/types/utils/formatSystemMetricName.d.ts

```diff
@@ -1,10 +1,58 @@
 export type systemMetricsDictType = {
-  __system__cpu: string;
-  __system__p_memory_percent: string;
-  __system__memory_percent: string;
-  __system__disk_percent: string;
-  __system__gpu: string;
-  __system__gpu_memory_percent: string;
-  __system__gpu_power_watts: string;
-  __system__gpu_temp: string;
-};
+  'system/cpu_utilization_percentage': string;
+  'system/disk_available_megabytes': string;
+  'system/disk_usage_megabytes': string;
+  'system/disk_usage_percentage': string;
+  'system/gpu_0_memory_usage_megabytes': string;
```

Comment:
Same comment re: tidying this w/an iterator...

Reply:
same reply as above: if there is a way to do this cleanly in typescript, I'll take it!